README.md

```diff
@@ -352,12 +352,14 @@ A `load_balancer` block supports the following:
 * `port` - (Optional) The port of the listener. Defaults to 443.
 
 * `container_name` - (Required) The name of the container to associate
-with the load balancer as it appears in the container definition (by
-default, `containers.json`).
+with the load balancer as it appears in the container definition
+(by default, `containers.json`). Defaults to the value specified
```

Comment:
This is false as the code is written.

Reply:
Directions unclear. What should the change be to reflect reality? Remove `container_name` variable from the `load_balancer` section?

Reply:
Please revert all changes to the README.md. I don't know how or why any of these changes are present. All the changes are incorrect. Revert the README.md to what is found in `master`.